packages/block-library/src/classic/edit.js

```diff
@@ -149,9 +154,13 @@ export default class ClassicEdit extends Component {
 						content: value,
 					} );
 				}
-			}, 250 )
+			} )
 		);
 
+		editor.on( 'remove', () => {
+			editor._isRemoved = true;
```

Comment:
Really minor suggestion, but maybe `_hasBeenRemoved`? Also, is there any property already in the TinyMCE API that would give us this information? (I don't know, I haven't found one yet)

Reply:
I've search but I didn't find neither a property or a function to get that information.